test/e2e/pipeline/pipeline_test.go

```diff
@@ -212,6 +223,27 @@ Waiting for logs to be available...
 		})
 	})
 
+	t.Run("Get list of Taskruns using field-selector from namespace  "+namespace, func(t *testing.T) {
```

Comment:
Could you move this to `test/e2e/task/start_test.go` since this is for pipeline tests?